test/jdk/java/lang/RuntimeTests/ExitLogging-FINER.properties

```diff
@@ -0,0 +1,8 @@
+############################################################
+#       java.lang. Runtime logging level to console to FINER
+############################################################
+
+handlers= java.util.logging.ConsoleHandler
+
+java.util.logging.ConsoleHandler.level = ALL
+java.lang.Runtime.level = FINER
```

Comment:
It is a waste of resources to test every combination of logger levels.
This is not a test of the filtering mechanism built into the logger; it is a test that the message is logged when the logger level is set to DEBUG or finer.
Only two test cases are needed for that.

Reply:
I see. This test only takes a very short time, but shouldn't the cases be added?

Reply:
Still no, starting a new Java runtime isn't that quick, and it brings no additional value.
All tests are run 10's of thousands of times in every release, adding a few seconds here and there add up.

Reply:
Thank you for the additional comment!
I understand that two test cases are sufficient as a general test for loggers.
However, the valid level of "System.exit and Runtime.exit debug logging" is determined by the following specific implementation:
https://github.com/openjdk/jdk/blob/41d4a0d7bdda2a96af1e7f549c05d99d68c040dc/src/java.base/share/classes/java/lang/Shutdown.java#L175
Does this code need to be tested exhaustively?